okapi/services/caches/save_user_coords/docs.xml

```diff
@@ -0,0 +1,30 @@
+<xml>
+    <brief>Update personal coordinates of a geocache</brief>
+    <issue-id>629</issue-id>
+    <desc>
+        <p>This method allows your users to to change the content of the user's
+        personal cache coordinates from one value to the other value.</p>
```

Comment:
How about this:

This method allows your users to update the coordinates of their personal geocache coordinates.

Reply:
good suggestion, will take it.